pymysql/connections.py

```diff
@@ -17,11 +17,12 @@
 import warnings
 
 from .charset import MBLENGTH, charset_by_name, charset_by_id
-from .cursors import Cursor
 from .constants import CLIENT, COMMAND, FIELD_TYPE, SERVER_STATUS
-from .util import byte2int, int2byte
 from .converters import (
     escape_item, encoders, decoders, escape_string, through)
+from .cursors import Cursor
+from .parsers import DefaultFileParser
```

Comment:
`parsers` has too wide meanings.
Parsing option file, SQL and packet is completely different concern.
How about `pymysql.optionfile.Parser` ?